config/default.yml

```diff
@@ -224,3 +228,7 @@ RSpec/VerifiedDoubles:
   Description: Prefer using verifying doubles over normal doubles.
   Enabled: true
   IgnoreSymbolicNames: false
+
+FactoryGirl/DynamicAttributeDefinedStatically:
```

Comment:
I just wonder if this shouldn't be under `RSpec/` namespace as well, e.g. does it work right now